virtcontainers/sandbox.go

```diff
@@ -574,8 +572,20 @@ func newSandbox(ctx context.Context, sandboxConfig SandboxConfig, factory Factor
 		}
```

Comment:
nit: the defer function should be combined with the above one.

Reply:
Yep, nice catch!